src/extensions/ExtensionManager.as

```diff
@@ -212,6 +212,10 @@ public class ExtensionManager {
 					// 'r' is reporter, 'R' is requester, and 'rR' is a reporter forced to act as a requester.
 					spec[0] = 'rR';
 				}
+				if(spec[0] == 'h') {
+					// same as above, but force to use async hat blocks
+					spec[0] = 'hH';
+				}
```

Comment:
I can't find the issue either, but the attempted fix was previously noted in the (now obsolete) #756